src/test/java/seedu/address/testutil/AssignmentBuilder.java

```diff
@@ -79,6 +84,7 @@ public AssignmentBuilder withDeadline(String deadline) {
 
     /**
      * Sets the {@code Remind} of the {@code Assignment} that we are building.
+     *
      * @return
      */
     public AssignmentBuilder withRemindersSet() {
```

Comment:
Oops must have forgotten to delete the empty return tag when I created this method. You can just delete the @return tag since the other methods don't seem to have the @return tag :)